### DISCUSSION TITLE
Search in pod logs

### DISCUSSION THREAD

**Discussion by savbran** [podman-desktop/podman-desktop#9725] [General]
Hello folks, from podman-desktop is there the possibility to not only view a running pod log but also to search text in it?
I'm referring to the Pods > Pod Details > Logs tab

![podman-desktop-log-tab](https://github.com/user-attachments/assets/2e6fb09c-e51a-4a60-b6e6-0ac541ebb9af)

**Comment by WalterClementsJr** [comment]
It seems to be available, as seen from the [docs](https://podman-desktop.io/docs/containers/viewing-container-logs).

But I can't use it either. I'm trying both Docker Desktop and Podman Desktop at work to compare them.
